src/mobile.wiki

```diff
@@ -63,10 +63,16 @@ software.
 
 For all mobile platforms:
 
-* Install {{{npm}}} (on Debian and Ubuntu, {{{sudo apt-get install npm}}}).
+* Install {{{NodeJS}}} and {{{NPM}}} (see
+  [[https://nodejs.org/en/download/package-manager/|the official website]]). If
+  you have some issues with your package manager, you can also use
+  [[https://github.com/creationix/nvm|nvm]].
```

Comment:
The official website encourages bad practices like piping `curl $RANDOM_URL` to `sudo bash` and relies on repositories of dubious quality. The Node ecosystem we know and love.

So I much prefer providing a shortcut for the most common setup (Debian/Ubuntu), and letting users of the other distributions and the *BSDs adapt. They are smart enough :).